cdi/src/test/java/org/infinispan/cdi/test/event/CacheEventTest.java

```diff
@@ -155,7 +155,7 @@ public void testFiringEntryModifiedEventWhenModifyingEntryInCache() throws Excep
 
    public void testFiringEntryInvalidatedWhenUsingCacheNotifier() throws Exception {
       //when
-      cache1Notifier.notifyCacheEntryInvalidated("pete", "Edinburgh", true, invocationContext, null);
+      cache1Notifier.notifyCacheEntryInvalidated("pete", "Edinburgh", null, true, invocationContext, null);
```

Comment:
True, I could refactor that out. The problem was that notifier was reading some values from InvocationContext, while simple cache uses a dummy-one, so I've changed this to provide the values directly.